libraries/helpers.rb

```diff
@@ -600,3 +600,16 @@ def format_directory(dir)
   format_dir = "/#{format_dir}" unless format_dir.start_with?('/')
   format_dir
 end
+
+def parse_arn(arn_string)
+  parts = arn_string.nil? ? [] : arn_string.split(':', 6)
```

Comment:
let's add an example of the format here as well. 
Question: Is this vaild for all the ARNs ?

Reply:
Adding comments on the format in next revision.
Yes, the vcode covers alla types of ARN. It's actually the same logic implemented by the AWS SDK for Ruby v3.